Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -2571,9 +2604,51 @@ - (void)tappedInvalidIdentityKeyErrorMessage:(TSInvalidIdentityKeyErrorMessage *
     [self presentViewController:actionSheetController animated:YES completion:nil];
 }
 
-- (void)tappedUnknownContactBlockOfferMessage:(OWSUnknownContactBlockOfferMessage *)errorMessage
+- (void)handleCallTap:(TSCall *)call
```

Comment:
This is just a method reordering in order to group the "handle tap in contact offers" methods together.